File: Lab_02/src/KI303/Kilchytska/Lab2/ControlButtons.java
package KI303.Kilchytska.Lab2;

/**
 * Клас ControlButtons описує кнопки керування монітором.
 */
public class ControlButtons {
    private int brightness;
    private int contrast;

    public ControlButtons() {
        brightness = 50;
        contrast = 50;
    }

    public void increaseBrightness() {
        if (brightness < 100) brightness++;
    }

    public void decreaseBrightness() {
        if (brightness > 0) brightness--;
    }

    public void increaseContrast() {
        if (contrast < 100) contrast++;
    }

    public void decreaseContrast() {
        if (contrast > 0) contrast--;
    }

    public int getBrightness() {
        return brightness;
    }

    public int getContrast() {
        return contrast;
    }

    @Override
    public String toString() {
        return "Brightness=" + brightness + ", Contrast=" + contrast;
    }
}

File: Lab_09/monitor_package/control_buttons.py
# monitor_package/control_buttons.py

class ControlButtons:
    """Клас ControlButtons описує кнопки керування монітором."""

    def __init__(self):
        """Конструктор для ініціалізації кнопок керування."""
        self.brightness = 50  # Початкова яскравість
        self.contrast = 50  # Початковий контраст

    def increase_brightness(self):
        """Збільшує яскравість, якщо вона менша за 100."""
        if self.brightness < 100:
            self.brightness += 1

    def decrease_brightness(self):
        """Зменшує яскравість, якщо вона більша за 0."""
        if self.brightness > 0:
            self.brightness -= 1

    def increase_contrast(self):
        """Збільшує контраст, якщо він менший за 100."""
        if self.contrast < 100:
            self.contrast += 1

    def decrease_contrast(self):
        """Зменшує контраст, якщо він більший за 0."""
        if self.contrast > 0:
            self.contrast -= 1

    def __str__(self):
        """Повертає рядкове представлення об'єкта."""
        return f'Brightness={self.brightness}, Contrast={self.contrast}'